### PR TITLE
Forms for user account editing & lockbox partner edits

### PR DESCRIPTION
**Pull request by arches**
## Changelog
- Ability for lockbox partners to edit their contact info
- Ability for users to change their password (but nothing else about their account)
- Update nav to include links for these actions
- Update nav to include partner contact info when logged in as a partner

## Link to issue:  
Fixes issue #196 

## Steps for QA/Special Notes:
- N/A

## Relevant Screenshots: 

New nav for partners:

LockboxRails

## Are you ready for review?:

- [x] Added relevant tests
- [x] Linked PR to the issue
- [x] Added notes for QA/special notes
- [x] Added relevant screenshots

### CODE REVIEW

File: app/views/layouts/_nav_bar.html.erb [hunk 1]
```diff
@@ -19,13 +19,19 @@
...
         <p>Quick Actions</p>
         <p><a href="<%= lockbox_partners_path %>">View all lockboxes</a></p>
         <p><a href="<%= support_requests_new_path %>">File a support request</a></p>
         <p><a href="<%= new_lockbox_partner_path %>">Add a new lockbox partner</a></p>
+      <% elsif lp = current_user.lockbox_partner %>
+        <h2><%= lp.name %></h2>
+        <p><%= lp.street_address %></p>
+        <p><%= lp.city %>, <%= lp.state %> <%= lp.zip_code %></p>
+        <p><%= lp.phone_number %></p>
+        <p><%= link_to "Edit Contact Information", edit_lockbox_partner_path(lp), class: "edit_contact_info btn btn-primary btn-invert" %></p>
```
Comment (misternu) [inline app/views/layouts/_nav_bar.html.erb]: potential conflicts between this pr and @BintLopez 's fix pw reset pr, I'm not git tho so I'm only guessing

Comment (misternu) [reply app/views/layouts/_nav_bar.html.erb]: #233

**Comment by misternu** [comment]
looks good so far, will re-review after nicole's merge

**Comment by BintLopez** [comment]
`CATS CLAWNIC` 😹 💯

File: app/views/layouts/_nav_bar.html.erb (hunk 1 above)
Comment (BintLopez) [reply app/views/layouts/_nav_bar.html.erb]: Let's let this merge first and then I'll fix any conflicts on my end since it's a smaller PR

**Pull request merged**